coldfront/core/utils/mail.py

```diff
@@ -102,7 +102,7 @@ def send_allocation_admin_email(allocation_obj, subject, template_name, url_path
     ctx = email_template_context()
     ctx['pi'] = pi_name
     ctx['resource'] = resource_name
-    ctx['url'] = url
+    ctx['allocation_url'] = url
```

Comment:
This function is used by lots of other templates. So you can't change the key here. Keep this as `ctx['url']` and you just need to update your new templates to use `{{url}}` instead of `{{allocation_url}}`